pkg/datapath/loader/base.go

```diff
@@ -30,6 +31,7 @@ import (
 	"github.com/cilium/cilium/pkg/datapath/alignchecker"
 	"github.com/cilium/cilium/pkg/datapath/linux/linux_defaults"
 	"github.com/cilium/cilium/pkg/datapath/linux/route"
+	linuxrouting "github.com/cilium/cilium/pkg/datapath/linux/routing"
```

Comment:
Looking at this again, now that we've shifted the actual route table specifics into this package I'm fine with this import. We clearly already directly import other packages under `datapath/linux` so if we want to improve the abstractions used in this area of the code more, then I think we should think about that outside of the context of this fix that we plan to backport.